src/html/preview.html

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
```

Comment:
Does it make sense to include this as the default html given it will be written over immediately?

Reply:
Yes—the page that gets loaded into the preview frame should still be valid HTML (and the doctype is particularly important). You _can_ skip the `<head>` and `<body>` tags, though.